src/Paket.Core/Dependencies/DependenciesFileParser.fs

```diff
@@ -43,19 +43,13 @@ module DependenciesFileParser =
                 match texts |> List.filter ((<>) "") with
                 | [] -> 
                     versions
-                    |> List.collect (fun version -> 
-                        match version.PreRelease with 
-                        | Some x -> [x.Name]
-                        | _ -> [])
+                    |> List.collect (function { PreRelease = Some x } -> [x.Name] | _ -> [])
```

Comment:
Probably not for most paket contributors, but in general I'm against function keyword since it's one more concept to learn and it's not needed at all